package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cplace/cli",
-  "version": "0.19.20",
+  "version": "0.20.0",
```

This is a breaking change and I would suggest to bump the major version
```suggestion
  "version": "1.0.0",
```

this will allow and simplify to maintain fixes and even possibly updating the '0' series and have minor version changes.